source/extensions/tracers/opentelemetry/samplers/dynatrace/dynatrace_sampler.cc

```diff
@@ -73,6 +72,26 @@ class DynatraceTag {
   uint32_t path_info_;
 };
 
+// add Dynatrace specific span attributes
+void addSamplingAttributes(uint32_t sampling_exponent,
+                           std::map<std::string, std::string>& attributes) {
+
+  const auto multiplicity = SamplingState::toMultiplicity(sampling_exponent);
+  // The denominator of the sampling ratio. If for example the Dynatrace OneAgent samples with a
+  // probability the value of supportability.atm_sampling_ratio would be 16
+  // ratio is also called multiplicity
```

Comment:
```suggestion
  // . Note: Ratio is also known as multiplicity.
```

I think this is just a "note" right? I'd start this in its own sentence, or add a final . before it

Reply:
I partially copied the description from https://bitbucket.lab.dynatrace.org/projects/DEUS/repos/semantic-dictionary/browse/doc/fields/supportability.md
Obviously, I removed essential parts. I modified the comment.